Source/lighting.cpp

```diff
@@ -1140,48 +1141,50 @@ void ChangeVisionRadius(int id, int r)
 void ChangeVisionXY(int id, Point position)
 {
 	for (int i = 0; i < VisionCount; i++) {
-		if (VisionList[i]._lid == id) {
-			VisionList[i]._lunflag = true;
-			VisionList[i].position.old = VisionList[i].position.tile;
-			VisionList[i].oldRadius = VisionList[i]._lradius;
-			VisionList[i].position.tile = position;
+		auto &vision = VisionList[i];
+		if (vision._lid == id) {
+			vision._lunflag = true;
+			vision.position.old = vision.position.tile;
+			vision.oldRadius = vision._lradius;
+			vision.position.tile = position;
 			dovision = true;
 		}
 	}
 }
 
 void ProcessVisionList()
 {
+	auto &vision = VisionList;

```

Comment:
Aliasing the global it self doesn't really help or solve anything.

Reply:
still, I like this change at it will make passing visionlist as function argument a bit easier

Reply:
Well, it will be irrelevant once the the function is cleaned up properly I think.